src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -149,18 +203,21 @@ const Conversation: React.FC<ConversationProps> = props => {
               {isOfferable && <BuyerGuaranteeMessage />}
               {inquiryItemBox}
               <Waypoint onEnter={loadMore} />
-              {fetchingMore ? (
-                <SpinnerContainer>
-                  <Spinner />
-                </SpinnerContainer>
-              ) : null}
+              {fetchingMore ? <Loading /> : null}
```

Comment:
I refactored this, initially, to use at the bottom as well but opted against it. Will probs revert this.